coral/src/app/features/topics/BrowseTopics.tsx

```diff
@@ -33,7 +33,7 @@ function BrowseTopics() {
   } = useGetTopics({
     currentPage: page,
     environment,
-    teamName: team,
+    teamName: team !== ALL_TEAMS_VALUE ? team : undefined,
```

Comment:
What do you think of changing that to 
`...(team !== ALL_TEAMS_VALUE && { teamName: team }),`?

I try to avoid setting `undefined` anywhere to make debugging easier in potential errors. 
That way, we would only set and pass the property `teamName` if it is not `ALL_TEAMS_VALUE`

Reply:
I do not have strong preference either way, so I changed this according yo your suggestion. Out of curiosity, how does it make debugging easier? From a pure performance point of view, this suggestion feels like a heavier operation as it merges objects. instead of simple property assignment. However, usually the readability is more important than performance.